components/licensor/ee/pkg/licensor/licensor.go

```diff
@@ -53,6 +53,9 @@ type LicensePayload struct {
 
 	// Seats == 0 means there's no seat limit
 	Seats int `json:"seats"`
+
+	// CustomerID is used to identify installations in installation analytics
+	CustomerID string `json:"customerID"`
```

Comment:
There are two hard problems in computer science; naming, cache invalidation, and off by one errors.

Within Replicated we have a custom `levelId` field; at present I'm naming this field `customerId` to match that pattern. However within this implementation we use `customerID` with an upper cased D. If anyone has strong opinions I'm happy to adjust this.